include/mbedtls/pk.h

```diff
@@ -349,6 +469,15 @@ int mbedtls_pk_sign( mbedtls_pk_context *ctx, mbedtls_md_type_t md_alg,
              unsigned char *sig, size_t *sig_len,
              int (*f_rng)(void *, unsigned char *, size_t), void *p_rng );
 
+/**
+ * \brief           Calculate the size of a signature made with this key.
```

Comment:
Suggestion: `Provide an upper bound for the size of a signature made with this key.`

Reply:
Done